db/errors.go

```diff
@@ -3,7 +3,7 @@ package db
 import (
 	"strings"
 
-	mgo "gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2"
```

Comment:
revert this.

Reply:
I will but I don't really see why we specify it if `mgo.whateverFunction` still works, and many files have removed this by now.